.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 // Place your settings in this file to overwrite default and user settings.
 {
     "files.exclude": {
-        "out": true, // set this to true to hide the "out" folder with the compiled JS files
+        "out": false, // set this to true to hide the "out" folder with the compiled JS files
```

Comment:
Please can we change this back.

Reply:
Not sure why this is changed. I don't recall making this change. Might have been in the original bits you wrote. I'll change it back.